modules/understanding-upgrade-channels.adoc

```diff
@@ -8,16 +8,43 @@
 [id="understanding-upgrade-channels_{context}"]
 
 = Upgrade channels and release paths
-Cluster administrators can configure the upgrade channel from the web console.
 
+ifndef::openshift-origin[]
+{product-title} {product-version} offers the following upgrade channels:
+
+* `candidate-{product-version}`
+* `fast-{product-version}`
+* `stable-{product-version}`
+* `eus-4.y` (only when running an even-numbered 4.y cluster release, like 4.10)
+
+If you do not want the Cluster Version Operator to fetch available updates from the upgrade recommendation service, you can use the `oc adm upgrade channel` command in the OpenShift CLI to configure an empty channel. This configuration can be helpful if, for example, a cluster has restricted network access and there is no local, reachable upgrade recommendation service.
+
+[WARNING]
+====
+Red Hat recommends upgrading to versions suggested by Openshift Update Service only. For minor version upgrade, versions must be contiguous. Red Hat does not test upgrades to noncontiguous versions and cannot guarantee compatibility with earlier versions.
+====
+
+endif::openshift-origin[]
+ifdef::openshift-origin[]
+{product-title} {product-version} offers the following upgrade channel:
+
+* `stable-4`
+
+endif::openshift-origin[]
 
 [id="candidate-version-channel_{context}"]
 == candidate-{product-version} channel
 
-The `candidate-{product-version}` channel contains candidate builds for a z-stream ({product-version}.z) and previous minor version releases. Release candidates contain all the features of the product but are not supported. Use release candidate versions to test feature acceptance and assist in qualifying the next version of {product-title}. A release candidate is any build that is available in the candidate channel, including ones that do not contain link:https://semver.org/spec/v2.0.0.html#spec-item-9[a pre-release version] such as `-rc` in their names. After a version is available in the candidate channel, it goes through more quality checks. If it meets the quality standard, it is promoted to the `fast-{product-version}` or `stable-{product-version}` channels. Because of this strategy, if a specific release is available in both the `candidate-{product-version}` channel and in the `fast-{product-version}` or `stable-{product-version}` channels, it is a Red Hat-supported version. The `candidate-{product-version}` channel can include release versions from which there are no recommended updates in any channel.
+The `candidate-{product-version}` channel contains pre-release channels with alpha, beta, and release candidate builds and versions. The `candidate` channel contains candidate builds for a z-stream and previous minor version releases.
```

Comment:
Any reason you have remove these texts 
```
After a version is available in the candidate channel, it goes through more quality checks. If it meets the quality standard, it is promoted to the `fast-{product-version}` or `stable-{product-version}` channels. Because of this strategy, if a specific release is available in both the `candidate-{product-version}` channel and in the `fast-{product-version}` or `stable-{product-version}` channels, it is a Red Hat-supported version. The `candidate-{product-version}` channel can include release versions from which there are no recommended updates in any channel.
```

Reply:
"pre-release" has [a specific SemVer definition][1] and is sufficient to cover our feature candidates (e.g. 4.9.0-fc.0), release candidates (e.g. 4.9.0-fc.1), and hotfixes (e.g. 4.5.0-0.hotfix-2020-08-24-185832).  And some of the wording you're dropping was trying to explain things like 4.9.0 not being officially supported while it was in `candidate-4.9` (openshift/cincinnati-graph-data#1129) but before it had entered `fast-4.9` (https://github.com/openshift/cincinnati-graph-data#1133) , etc.  So basically I like a lot of the text you're dropping, and if there are issues with any of that text, I think we should lay out pros and cons and points that we're trying to convey and sections that are causing confusion, etc.

[1]: https://semver.org/spec/v2.0.0.html#spec-item-9

Reply:
@LalatenduMohanty @wking - Most of the updates are being done to make this section more understandable by someone who is new to OpenShift and needs to understand updates. It is quite confusing as it is laid out right now. The current text that I submitted in this PR is a suggestion based on the "How OpenShift Updates Work" RHU course. I'm obviously open to suggestions and clarifications as needed for this section. I just think we need to lay out in such a way that it is easy to understand.